help-wanted.md

```diff
@@ -0,0 +1,126 @@
+
+- **Low Barrier to Entry**
+
+    It should be tractable for new contributors.
+    Documentation on how that type of change should be made should already exist.
+
+- **Clear Task**
+
+    The task is agreed upon and does not require further discussions in the community.
+    Call out if that area of code is untested and requires new fixtures.
+
+    API / CLI behavior is decided and included in the OP issue,
+    for example: *"The new command syntax is `foo bar NAME [--unFoo] [--timeout 5m]`"*,
+    with expected validations called out.
+
+- **Goldilocks priority**
+
+    Not too high that a core contributor should do it,
+    but not too low that it isn't useful enough for a core contributor to spend time to review it,
+    answer questions, help get it into a release, etc.
+
+- **Up-To-Date**
+
+    Often these issues become obsolete and have already been done,
+    are no longer desired, no longer make sense, have changed priority or difficulty, etc.
+    The maintainers of Traefik regularly groom and manage these issues to ensure they remain relevant.
+
+## Good First Issue
+
+Items marked with the `contributor/good-first-issue` label are intended for *first-time contributors*.
+It indicates that maintainers will keep an eye out for these pull requests
+and shepherd it through our processes.
+
+**New contributors should not be left to find an approver, ping for reviews,
+or identify that their build failed due to a flake.**
+
+This makes new contributors feel welcome and valued,
+further assuring them that they will have an extra level of help with their first contribution.
+
+After a contributor has successfully completed 1-2 `contributor/good-first-issue`'s, 
+they should be ready to move on to `contributor/help-wanted` items,
+saving remaining `contributor/good-first-issue`'s for other new contributors.
+
+These items need to ensure that they follow the guidelines for `contributor/good-first-issue` labels (above)
+in addition to meeting the following criteria:
+
+- **No Barrier to Entry**
+
+    The task is something that a new contributor can tackle without advanced setup or domain knowledge.
+
+- **Solution Explained**
+
+    The recommended solution is clearly described in the issue.
+
+- **Provides Context**
+
+    If background knowledge is required, 
+    this should be explicitly mentioned and a list of suggested readings included.
+
+- **Gives Examples**
+
+    Link to examples of similar implementations so new contributors have a reference guide for their changes.
+
+- **Identifies Relevant Code**
+
+    The relevant code and tests to be changed should be linked in the issue.
+
+- **Ready to Test**
+
+    There should be existing tests that can be modified, or existing test cases fit to be copied.
+    If the area of code doesn't have tests, before labeling the issue, add a test fixture.
+    This prep often makes a great `contributor/good-first-issue` task!
+
+# Suggestions
```

Comment:
All of this part concerns communication management and does not seem relevant here.

Reply:
I'm still recovering from illness, so I'm still a bit hazy -- I'll probably hold off on conversating on the other points because I'm unclear on if there are any action items for them.

However, for this one, I believe the goal is to provide communication management advice for external maintainers and contributors. We could put this somewhere in notion, or elsewhere, but it seems relevant here if our goal is to eventually empower members of the community to step up and engage in these activities. WDYT?